rust/src/helpers/array.rs

```diff
@@ -23,14 +23,28 @@ where
     }
 }
 
+#[allow(clippy::range_plus_one, clippy::too_many_lines)]
```

Comment:
Why do we need the range plus one?

Reply:
unfortunate that you have to exclude the entire method and you can just exclude the instance

Reply:
You can but that is marked "experimental".  I checked. :)

Reply:
(Attributes on expressions)